appveyor.yml

```diff
@@ -1,3 +1,19 @@
+#
+# Copyright (C) 2016 Red Hat, Inc.
```

Comment:
I think you usually use the year of inception? Or else we need to update all the headers every year? I don't feel strongly either way though. Just like to keep the noise low.